src/constant.h

```diff
@@ -7,594 +7,598 @@
+constexpr int32_t STORE_INVEN_MAX = 24; // Max number of discrete objects in inventory
+constexpr int32_t STORE_CHOICES = 26; // NUMBER of items to choose stock from
+constexpr int32_t STORE_MAX_INVEN = 18; // Max diff objects in stock for auto buy
+constexpr int32_t STORE_MIN_INVEN = 10; // Min diff objects in stock for auto sell
+constexpr int32_t STORE_TURN_AROUND = 9; // Amount of buying and selling normally
+constexpr int32_t COST_ADJ = 100; // Adjust prices for buying and selling
 
 // Treasure constants
-#define INVEN_ARRAY_SIZE         34   // Size of inventory array(Do not change)
-#define MAX_OBJ_LEVEL            50   // Maximum level of magic in dungeon
-#define OBJ_GREAT                12   // 1/n Chance of item being a Great Item
+constexpr int32_t INVEN_ARRAY_SIZE = 34; // Size of inventory array(Do not change)
+constexpr int32_t MAX_OBJ_LEVEL = 50; // Maximum level of magic in dungeon
+constexpr int32_t OBJ_GREAT = 12; // = 1/n Chance of item being a Great Item
 
 // Note that the following constants are all related, if you change one, you
 // must also change all succeeding ones. Also, player_base_provisions[] and
 // store_choices[] may also have to be changed.
-#define MAX_OBJECTS             420   // Number of objects for universe
-#define MAX_DUNGEON_OBJ         344   // Number of dungeon objects
-#define OBJ_OPEN_DOOR           367
-#define OBJ_CLOSED_DOOR         368
-#define OBJ_SECRET_DOOR         369
-#define OBJ_UP_STAIR            370
-#define OBJ_DOWN_STAIR          371
-#define OBJ_STORE_DOOR          372
-#define OBJ_TRAP_LIST           378
-#define OBJ_RUBBLE              396
-#define OBJ_MUSH                397
-#define OBJ_SCARE_MON           398
-#define OBJ_GOLD_LIST           399
-#define OBJ_NOTHING             417
-#define OBJ_RUINED_CHEST        418
-#define OBJ_WIZARD              419
-#define OBJECT_IDENT_SIZE       448   // 7*64, see object_offset() in desc.c, could be MAX_OBJECTS o_o() rewritten
-#define MAX_GOLD                 18   // Number of different types of gold
-
-// with MAX_TALLOC 150, it is possible to get compacting objects during
+constexpr int32_t MAX_OBJECTS = 420; // Number of objects for universe
+constexpr int32_t MAX_DUNGEON_OBJ = 344; // Number of dungeon objects
+constexpr int32_t OBJ_OPEN_DOOR = 367;
+constexpr int32_t OBJ_CLOSED_DOOR = 368;
+constexpr int32_t OBJ_SECRET_DOOR = 369;
+constexpr int32_t OBJ_UP_STAIR = 370;
+constexpr int32_t OBJ_DOWN_STAIR = 371;
+constexpr int32_t OBJ_STORE_DOOR = 372;
+constexpr int32_t OBJ_TRAP_LIST = 378;
+constexpr int32_t OBJ_RUBBLE = 396;
+constexpr int32_t OBJ_MUSH = 397;
+constexpr int32_t OBJ_SCARE_MON = 398;
+constexpr int32_t OBJ_GOLD_LIST = 399;
+constexpr int32_t OBJ_NOTHING = 417;
+constexpr int32_t OBJ_RUINED_CHEST = 418;
+constexpr int32_t OBJ_WIZARD = 419;
+constexpr int32_t OBJECT_IDENT_SIZE = 448; // = 7*64, see object_offset() in desc.c, could be MAX_OBJECTS o_o() rewritten
```

Comment:
Search/replace error: `// 7*64, see object_offset() in desc.c, could be MAX_OBJECTS o_o() rewritten`